NitroxModel/Packets/CyclopsChangeEngineMode.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class CyclopsChangeEngineMode : AuthenticatedPacket
+    {
+        public String Guid { get; set; }
```

Comment:
This is dangerous: you should remove the `set` functions entirely (not just `private`), as every other object can modify the data now, which is intended to be immutable.